Using-pkcs11-tool-and-OpenSSL.md

```diff
@@ -1,171 +1,241 @@
-This document was [initially](https://gist.github.com/Jakuje/5a993d2b2d8a9cac35203599e49e6831) created for myself to memorize many command line options and because it was very handy for debugging to issue single operation to the PKCS#11 module for debugging. But it can be also useful for others who are interested in scripting these tasks or who are just curious what they can do with their new smart card.
+# Using `pkcs11-tool` and OpenSSL
```

Comment:
We also have examples in the manual page for pkcs11-tool or pkcs15-crypt. Maybe some of the examples here could be copied to there as well.